### PR TITLE
Adding handwired Maple60 keyboard

### PR DESCRIPTION
**Pull request by Toofty5**
## Description

Personal handwired 60% keyboard

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/handwired/maple60/keymaps/Toofty5/keymap.c [hunk 1]
```diff
@@ -0,0 +1,38 @@
+/* Copyright 2022 Jay(Toofty5)
```
Comment (drashna) [inline keyboards/handwired/maple60/keymaps/Toofty5/keymap.c:1]: please rename this folder to be lower case.

File: keyboards/handwired/maple60/config.h [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0xFEED
+#define PRODUCT_ID      0x0000
+#define DEVICE_VER      0x0001
+#define MANUFACTURER    "Jay"
+#define PRODUCT         "handwired/maple60"
```
Comment (drashna) [inline keyboards/handwired/maple60/config.h:26]:
```suggestion
#define MANUFACTURER    Jay
#define PRODUCT         handwired/maple60
```

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/handwired/maple60/readme.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
+# maple60
+
+![maple60](imgur.com image replace me!)
+
+*A short description of the keyboard/project*
+
+* Keyboard Maintainer: [Jay](https://github.com/Toofty5)
+* Hardware Supported: Custom
+* Hardware Availability: Custom
```
Comment (drashna) [inline keyboards/handwired/maple60/readme.md:9]: these need to be filled out

Comment (Toofty5) [reply keyboards/handwired/maple60/readme.md]: Added description, etc with photos

File: keyboards/handwired/maple60/rules.mk [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+#
+BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
+MOUSEKEY_ENABLE = no       # Mouse keys
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = yes        # Console for debug
+COMMAND_ENABLE = yes        # Commands for debug and configuration
+NKRO_ENABLE = no            # Enable N-Key Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
+BLUETOOTH_ENABLE = no       # Enable Bluetooth
```
Comment (drashna) [inline keyboards/handwired/maple60/rules.mk:18]:
```suggestion
```

File: keyboards/handwired/maple60/rules.mk [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+MCU = atmega32u4
+
+# Bootloader selection
+BOOTLOADER = atmel-dfu
+
+# Build Options
+#   change yes to no to disable
+#
+BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
+MOUSEKEY_ENABLE = no       # Mouse keys
```
Comment (drashna) [inline keyboards/handwired/maple60/rules.mk:11]:
```suggestion
MOUSEKEY_ENABLE = no        # Mouse keys
```

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/handwired/maple60/readme.md [hunk 6]
```diff
@@ -0,0 +1,27 @@
+# maple60
+
+A handwired 60% keyboard with maple case.
+
+<img src="https://user-images.githubusercontent.com/6307033/151084291-9d7a800d-4619-462d-a24c-768d7e9213cc.jpg" width="200"> <img src="https://user-images.githubusercontent.com/6307033/151084297-91f25027-c272-4aa3-ad2b-c35896eb2a2d.jpg" width="200">
```
Comment (drashna) [inline keyboards/handwired/maple60/readme.md:5]: Please remove the html tags, and use markdown for the images (eg ``, and host the image on imgur.

**Review by drashna — APPROVED** [review]

File: keyboards/handwired/maple60/readme.md [hunk 7]
```diff
@@ -0,0 +1,28 @@
+# maple60
+
+A handwired 60% keyboard with maple case.
+
+![inside](https://i.imgur.com/mDJDwTt.jpg)
+![finished](https://i.imgur.com/gIoEg6c.jpg)
```
Comment (fauxpark) [inline keyboards/handwired/maple60/readme.md:6]:
```suggestion
![inside](https://i.imgur.com/mDJDwTth.jpg)
![finished](https://i.imgur.com/gIoEg6ch.jpg)
```

File: keyboards/handwired/maple60/readme.md [hunk 8]
```diff
@@ -0,0 +1,28 @@
...
+* Hardware Supported: [Elite C v4](https://keeb.io/products/elite-c-low-profile-version-usb-c-pro-micro-replacement-atmega32u4)
+* Hardware Availability: See Above
+
+Make example for this keyboard (after setting up your build environment):
+
+    make maple60:default
+
+Flashing example for this keyboard:
+
+    make maple60:default:flash
```
Comment (fauxpark) [inline keyboards/handwired/maple60/readme.md:18]:
```suggestion
    make handwired/maple60:default

Flashing example for this keyboard:

    make handwired/maple60:default:flash
```

File: keyboards/handwired/maple60/config.h [hunk 9]
```diff
@@ -0,0 +1,50 @@
...
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0xFEED
+#define PRODUCT_ID      0x0000
+#define DEVICE_VER      0x0001
+#define MANUFACTURER    Jay
+#define PRODUCT         handwired/maple60
```
Comment (fauxpark) [inline keyboards/handwired/maple60/config.h:26]:
```suggestion
#define PRODUCT         Maple60
```

File: keyboards/handwired/maple60/info.json [hunk 10]
```diff
@@ -0,0 +1,74 @@
+{
+    "keyboard_name": "handwired/maple60",
+    "url": "",
+    "maintainer": "Toofty5",
+    "layouts": {
+        "LAYOUT": {
```
Comment (fauxpark) [inline keyboards/handwired/maple60/info.json:6]:
```suggestion
        "LAYOUT_60_ansi": {
```

File: keyboards/handwired/maple60/keymaps/default/keymap.c [hunk 11]
```diff
@@ -0,0 +1,38 @@
...
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+    [0] = LAYOUT(
```
Comment (fauxpark) [inline keyboards/handwired/maple60/keymaps/default/keymap.c:21]:
```suggestion
    [0] = LAYOUT_60_ansi(
```

File: keyboards/handwired/maple60/keymaps/default/keymap.c [hunk 12]
```diff
@@ -0,0 +1,38 @@
...
+        KC_ESC,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_MINS, KC_EQL,  KC_BSPC, 
+        KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_LBRC, KC_RBRC, KC_BSLS, 
+        KC_CAPS, KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT, KC_ENT,  
+        KC_LSFT, KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, KC_LSFT, 
+        KC_LCTL, KC_LGUI, KC_LALT, KC_SPC,  KC_LALT, KC_RGUI, MO(1),   KC_LCTL  
+    ),
+
+    [1] = LAYOUT(
```
Comment (fauxpark) [inline keyboards/handwired/maple60/keymaps/default/keymap.c:29]:
```suggestion
    [1] = LAYOUT_60_ansi(
```

File: keyboards/handwired/maple60/keymaps/toofty5/keymap.c [hunk 13]
```diff
@@ -0,0 +1,38 @@
...
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+    [0] = LAYOUT(
```
Comment (fauxpark) [inline keyboards/handwired/maple60/keymaps/toofty5/keymap.c:21]:
```suggestion
    [0] = LAYOUT_60_ansi(
```

File: keyboards/handwired/maple60/keymaps/toofty5/keymap.c [hunk 14]
```diff
@@ -0,0 +1,38 @@
...
+        KC_GESC,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_MINS, KC_EQL,  KC_BSPC, 
+        KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_LBRC, KC_RBRC, KC_BSLS, 
+        KC_CAPS, KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT, KC_ENT,  
+        KC_LSFT, KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, KC_RSFT, 
+        KC_LCTL, KC_LGUI, KC_LALT, KC_SPC,  KC_RALT, KC_RGUI, MO(1),   KC_RCTL  
+    ),
+
+    [1] = LAYOUT(
```
Comment (fauxpark) [inline keyboards/handwired/maple60/keymaps/toofty5/keymap.c:29]:
```suggestion
    [1] = LAYOUT_60_ansi(
```

File: keyboards/handwired/maple60/maple60.h [hunk 15]
```diff
@@ -0,0 +1,38 @@
...
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "quantum.h"
+
+#define XXX KC_NO
+
+#define LAYOUT( \
```
Comment (fauxpark) [inline keyboards/handwired/maple60/maple60.h:23]:
```suggestion
#define LAYOUT_60_ansi( \
```

File: keyboards/handwired/maple60/rules.mk [hunk 16]
```diff
@@ -0,0 +1,21 @@
...
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = yes        # Console for debug
+COMMAND_ENABLE = yes        # Commands for debug and configuration
+NKRO_ENABLE = no            # Enable N-Key Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
+AUDIO_ENABLE = no           # Audio output
+
+# generated by KBFirmware JSON to QMK Parser
+# https://noroadsleft.github.io/kbf_qmk_converter/
```
Comment (fauxpark) [inline keyboards/handwired/maple60/rules.mk:21]:
```suggestion

LAYOUTS = 60_ansi
```

File: keyboards/handwired/maple60/maple60.h [hunk 17]
```diff
@@ -0,0 +1,38 @@
...
+    K00, K01, K02, K03, K04, K05, K06, K07, K08, K09, K0A, K0B, K0C, K0D, \
+    K10, K11, K12, K13, K14, K15, K16, K17, K18, K19, K1A, K1B, K1C, K1D, \
+    K20, K21, K22, K23, K24, K25, K26, K27, K28, K29, K2A, K2B, K2D, \
+    K30, K31, K32, K33, K34, K35, K36, K37, K38, K39, K3A, K3D, \
+    K40, K41, K42, K45, K49, K4A, K4C, K4D  \
```
Comment (fauxpark) [inline keyboards/handwired/maple60/maple60.h:28]:
```suggestion
    K00, K01, K02, K03, K04, K05, K06, K07, K08, K09, K0A, K0B, K0C, K0D, \
    K10, K11, K12, K13, K14, K15, K16, K17, K18, K19, K1A, K1B, K1C, K1D, \
    K20, K21, K22, K23, K24, K25, K26, K27, K28, K29, K2A, K2B,      K2D, \
    K30,      K31, K32, K33, K34, K35, K36, K37, K38, K39, K3A,      K3D, \
    K40, K41, K42,                K45,                K49, K4A, K4C, K4D \
```

File: keyboards/handwired/maple60/keymaps/default/keymap.c [hunk 18]
```diff
@@ -0,0 +1,38 @@
...
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+    [0] = LAYOUT(
+        KC_ESC,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_MINS, KC_EQL,  KC_BSPC, 
+        KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_LBRC, KC_RBRC, KC_BSLS, 
+        KC_CAPS, KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT, KC_ENT,  
+        KC_LSFT, KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, KC_LSFT, 
+        KC_LCTL, KC_LGUI, KC_LALT, KC_SPC,  KC_LALT, KC_RGUI, MO(1),   KC_LCTL
```
Comment (fauxpark) [inline keyboards/handwired/maple60/keymaps/default/keymap.c:26]:
```suggestion
        KC_ESC,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_MINS, KC_EQL,  KC_BSPC,
        KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_LBRC, KC_RBRC, KC_BSLS,
        KC_CAPS, KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT,          KC_ENT,
        KC_LSFT,          KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH,          KC_LSFT,
        KC_LCTL, KC_LGUI, KC_LALT,                            KC_SPC,                             KC_LALT, KC_RGUI, MO(1),   KC_LCTL
```

File: keyboards/handwired/maple60/keymaps/default/keymap.c [hunk 19]
```diff
@@ -0,0 +1,38 @@
...
+        KC_LSFT, KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, KC_LSFT, 
+        KC_LCTL, KC_LGUI, KC_LALT, KC_SPC,  KC_LALT, KC_RGUI, MO(1),   KC_LCTL  
+    ),
+
+    [1] = LAYOUT(
+        KC_GRV,  KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,  KC_DEL,  
+        _______, _______, _______,   _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, 
+        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, 
+        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, 
+        _______, _______, _______, _______, _______, _______, _______, _______
```
Comment (fauxpark) [inline keyboards/handwired/maple60/keymaps/default/keymap.c:34]:
```suggestion
        KC_GRV,  KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,  KC_DEL,
        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______,
        _______,          _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______,
        _______, _______, _______,                            _______,                            _______, _______, _______, _______
```

File: keyboards/handwired/maple60/keymaps/toofty5/keymap.c [hunk 20]
```diff
@@ -0,0 +1,38 @@
...
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+    [0] = LAYOUT(
+        KC_GESC,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_MINS, KC_EQL,  KC_BSPC, 
+        KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_LBRC, KC_RBRC, KC_BSLS, 
+        KC_CAPS, KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT, KC_ENT,  
+        KC_LSFT, KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, KC_RSFT, 
+        KC_LCTL, KC_LGUI, KC_LALT, KC_SPC,  KC_RALT, KC_RGUI, MO(1),   KC_RCTL
```
Comment (fauxpark) [inline keyboards/handwired/maple60/keymaps/toofty5/keymap.c:26]:
```suggestion
        KC_GESC, KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_MINS, KC_EQL,  KC_BSPC,
        KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_LBRC, KC_RBRC, KC_BSLS,
        KC_CAPS, KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT,          KC_ENT,
        KC_LSFT,          KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH,          KC_RSFT,
        KC_LCTL, KC_LGUI, KC_LALT,                            KC_SPC,                             KC_RALT, KC_RGUI, MO(1),   KC_RCTL
```

File: keyboards/handwired/maple60/keymaps/toofty5/keymap.c [hunk 21]
```diff
@@ -0,0 +1,38 @@
...
+        KC_LSFT, KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, KC_RSFT, 
+        KC_LCTL, KC_LGUI, KC_LALT, KC_SPC,  KC_RALT, KC_RGUI, MO(1),   KC_RCTL  
+    ),
+
+    [1] = LAYOUT(
+        KC_GRAVE,  KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,  KC_DEL,  
+        _______, LCTL(KC_LEFT), KC_UP, LCTL(KC_RGHT), _______, _______, _______, _______, _______, _______, _______, KC_VOLD, KC_VOLU, KC_MUTE, 
+        _______, KC_LEFT, KC_DOWN, KC_RGHT, _______, _______, _______, _______, _______, _______, _______, _______, _______, 
+        _______, _______, _______, _______, _______, _______, _______, _______, KC_LEFT, KC_RGHT, _______, _______, 
+        _______, _______, _______, _______, _______, _______, _______, _______
```
Comment (fauxpark) [inline keyboards/handwired/maple60/keymaps/toofty5/keymap.c:34]:
```suggestion
        KC_GRV,  KC_F1,      KC_F2,   KC_F3,      KC_F4,   KC_F5,   KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,  KC_DEL,
        _______, C(KC_LEFT), KC_UP,   C(KC_RGHT), _______, _______, _______, _______, _______, _______, _______, KC_VOLD, KC_VOLU, KC_MUTE,
        _______, KC_LEFT,    KC_DOWN, KC_RGHT,    _______, _______, _______, _______, _______, _______, _______, _______,          _______,
        _______,             _______, _______,    _______, _______, _______, _______, _______, KC_LEFT, KC_RGHT, _______,          _______,
        _______, _______,    _______,                               _______,                            _______, _______, _______, _______
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/handwired/maple60/config.h [hunk 22]
```diff
@@ -0,0 +1,50 @@
...
+ */
+#define MATRIX_ROW_PINS { F0, B6, B2, B3, B1 }
+#define MATRIX_COL_PINS { D3, D2, D1, D0, D4, C6, D7, E6, B4, B5, B7, D5, C7, F1 }
+
+#define DIODE_DIRECTION COL2ROW
+#define TAP_CODE_DELAY 10
+
+
+// generated by KBFirmware JSON to QMK Parser
+// https://noroadsleft.github.io/kbf_qmk_converter/
```
Comment (noroadsleft) [inline keyboards/handwired/maple60/config.h:50]:
```suggestion
```

File: keyboards/handwired/maple60/maple60.c [hunk 23]
```diff
@@ -0,0 +1,20 @@
...
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "maple60.h"
+
+// generated by KBFirmware JSON to QMK Parser
+// https://noroadsleft.github.io/kbf_qmk_converter/
```
Comment (noroadsleft) [inline keyboards/handwired/maple60/maple60.c:20]:
```suggestion
```

File: keyboards/handwired/maple60/maple60.h [hunk 24]
```diff
@@ -0,0 +1,38 @@
...
+) { \
+    { K00, K01, K02, K03, K04, K05, K06, K07, K08, K09, K0A, K0B, K0C, K0D }, \
+    { K10, K11, K12, K13, K14, K15, K16, K17, K18, K19, K1A, K1B, K1C, K1D }, \
+    { K20, K21, K22, K23, K24, K25, K26, K27, K28, K29, K2A, K2B, XXX, K2D }, \
+    { K30, K31, K32, K33, K34, K35, K36, K37, K38, K39, K3A, XXX, XXX, K3D }, \
+    { K40, K41, K42, XXX, XXX, K45, XXX, XXX, XXX, K49, K4A, XXX, K4C, K4D }, \
+}
+
+// generated by KBFirmware JSON to QMK Parser
+// https://noroadsleft.github.io/kbf_qmk_converter/
```
Comment (noroadsleft) [inline keyboards/handwired/maple60/maple60.h:38]:
```suggestion
```

File: keyboards/handwired/maple60/info.json [hunk 25]
```diff
@@ -0,0 +1,74 @@
...
+                { "label": "K42 (B1,D1)", "x": 2.5, "y": 4, "w": 1.25 },
+                { "label": "K45 (B1,C6)", "x": 3.75, "y": 4, "w": 6.25 },
+                { "label": "K49 (B1,B5)", "x": 10, "y": 4, "w": 1.25 },
+                { "label": "K4A (B1,B7)", "x": 11.25, "y": 4, "w": 1.25 },
+                { "label": "K4C (B1,C7)", "x": 12.5, "y": 4, "w": 1.25 },
+                { "label": "K4D (B1,F1)", "x": 13.75, "y": 4, "w": 1.25 }
+            ]
+        }
+    }
+    ,"meta": "https://noroadsleft.github.io/kbf_qmk_converter/"
```
Comment (noroadsleft) [inline keyboards/handwired/maple60/info.json:72]:
```suggestion
```

**Review by noroadsleft — CHANGES_REQUESTED** [review]

File: keyboards/handwired/maple60/readme.md [hunk 26]
```diff
@@ -0,0 +1,28 @@
...
+
+See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).
+
+## Bootloader
+
+Enter the bootloader in 3 ways:
+
+* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (usually the top left key or Escape) and plug in the keyboard
+* **Physical reset button**: Briefly press the button on the back of the PCB - some may have pads you must short instead
+* **Keycode in layout**: Press the key mapped to `RESET` if it is available
```
Comment (noroadsleft) [inline keyboards/handwired/maple60/readme.md:28]:
```suggestion
## Bootloader

Enter the bootloader in 3 ways:

* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (usually the top left key or Escape) and plug in the keyboard
* **Physical reset button**: Briefly press the button on the back of the PCB - some may have pads you must short instead
* **Keycode in layout**: Press the key mapped to `RESET` if it is available

See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).
```

**Review by noroadsleft — CHANGES_REQUESTED** [review]

**Pull request closed without merging**